com.unity.render-pipelines.universal/ShaderLibrary/UnityInput.hlsl

```diff
@@ -63,6 +63,9 @@ float4 _ScreenParams;
 // y = 2.0 ^ [Mip Bias]
 float2 _GlobalMipBias;
 
+// 1.0 if AlphaToMask is enabled and 0.0 otherwise
+float _AlphaToMaskEnabled;
```

Comment:
Is this matching with built-in? I think the UnityInput was for values from the Engine, or values that we setup but match built-in/engine.

Reply:
Hmmm, no it's not matching built-in. I was just looking for a place I could put a shader parameter where it'd be globally accessible. Is there a better place for this type of thing in URP?

Reply:
Looking at the UnityInput.hlsl, it looks like the "rule" has been broken already. 
UnityInput.hlsl was supposed to match 
https://docs.unity3d.com/2022.1/Documentation/Manual/SL-UnityShaderVariables.html
 as far as I know, but there's other stuff too like _GlobalMipBias.
 Perhaps, it's not an issue then.

I think the correct place would be Input.hlsl where all the URP constants are.